common/kubelet-wrapper

```
@@ -0,0 +1,67 @@
+#!/bin/bash -e
+# Wrapper for launching kubelet via docker.
+
+SYSTEMD_SERVICE_NAME="$1"; shift
```

If we do not specify any parameter for kubelet-wrapper, it silently fails. A little weird.
We should either error out or generate a systemd service unit name.

---

I agree, but on the other hand, this script is only called from one place with the correct parameter, and we are currently restricted in the size of `user data` on AWS to 16KB, so I tried to limit characters to the minimum.

---

I see. The limitation of 16KB is really a pain.